app/helpers/task_list_helper.rb

```diff
@@ -124,9 +124,12 @@ def row_helper(trainee, task)
       funding_active:
         {
           task_name: "Funding",
-          path: "#",
-          confirm_path: "#",
-          status: "not started",
+          path: edit_trainee_funding_training_initiative_path(trainee),
+          confirm_path: edit_trainee_funding_training_initiative_path(trainee),
+          status: ProgressService.call(
+            validator: Funding::TrainingInitiativesForm.new(trainee),
+            progress_value: trainee.progress.funding,
```

Comment:
@mooktakim - I've called this progress key `funding`. This will be needed for the confirmation controller.